### ISSUE TITLE
Universal vs MonoArm64

### ISSUE THREAD

**Issue by nightznero** [brave/brave-browser#32689]
Whats the differences if MONO64 uses both archs? I remember that a half year ago mono atm was literally mono but now....? It's mistake that nobody noticed yet or could anyone explain my why now mono have both archs If Universal is for that and also why the same libs in diff relase have different size?
Universal v8 almost 130mb while mono v8 is 106mb?
![20230902_231045](https://github.com/brave/brave-browser/assets/61750641/892162d4-81fc-48d9-a670-5fb1ce5dfe2a)

**Comment by libreom** [comment]
Can anyone answer now?

**Comment by Revival8697** [comment]
Checking [v1.74.50](https://github.com/brave/brave-browser/releases/tag/v1.74.50).
- [BraveMonoarm64.apk](https://github.com/brave/brave-browser/releases/download/v1.74.50/BraveMonoarm64.apk) includes both 64-bit and 32-bit. I don't think that is intended though.
- [Bravearm64Universal.apk](https://github.com/brave/brave-browser/releases/download/v1.74.50/Bravearm64Universal.apk) only includes 64-bit, enables [Isolated Splits](https://chromium.googlesource.com/chromium/src/+/main/docs/android_isolated_splits.md), which supposedly reduce RAM usage and speed up app launch.
- [BraveMonoarm64.aab](https://github.com/brave/brave-browser/releases/download/v1.74.50/BraveMonoarm64.aab) is the same as Bravearm64Universal.apk.

**Comment by LZeugirdor** [comment]
> Checking [v1.74.50](https://github.com/brave/brave-browser/releases/tag/v1.74.50):
> - [BraveMonoarm64.apk](https://github.com/brave/brave-browser/releases/download/v1.74.50/BraveMonoarm64.apk) includes both 64-bit and 32-bit. I don't think that is intended though.
> - [Bravearm64Universal.apk](https://github.com/brave/brave-browser/releases/download/v1.74.50/Bravearm64Universal.apk) only includes 64-bit, enables [Isolated Splits](https://chromium.googlesource.com/chromium/src/+/main/docs/android_isolated_splits.md), which supposedly reduce RAM usage and speed up app launch.
> - [BraveMonoarm64.aab](https://github.com/brave/brave-browser/releases/download/v1.74.50/BraveMonoarm64.aab) is the same as Bravearm64Universal.apk.

The logic is so weird, shouldn't mono arm only be arm 64 and universal have both binaries? That seems more sensible.

**Comment by Revival8697** [comment]
> The logic is so weird, shouldn't mono arm only be arm 64 and universal have both binaries? That seems more sensible.

Yes. But I don't understand what it really stand for and which context the devs use it in.

**Comment by Adhjie** [comment]
Do the brave team ever solve their github issue, im sorry to say this. but there's this old issue, and then the system-wide installation option for enterprise. 5k issues open. I don't understand. but it's nice to get bumped about this again.
until they answer, I guess.

edit 1:
even github cant show it all, 9k+*

edit 2:
one thing for certain tho, they dont package their software like other OS devs. where there are option for arm64, arm7 armeabi, x86, x86_64 (huge apk, not good for people with limited internet quota, and also a waste of space)
Theres also option for f-droid or other app stores, but i prefer Obtainium way of obtaining apps, straight from the release page of the dev app repo.

**Comment by nightznero** [comment]
> Do the brave team ever solve their github issue, im sorry to say this. but there's this old issue

if its not reported on their forum i guess they dont mind

> but it's nice to get bumped about this again. until they answer, I guess.

Take a nap,we gonna wait for it cowboy

**Comment by bushman74** [comment]
They don't seem to care about anything at all. Even if you make PRs, although you shouldn't as a mere consumer.

Any update on this preposterous issue of theirs?

**Comment by LogicalError007** [comment]
This issue is 2 years old? 
I decided to install all apps that are available on F-Droid not from Play Store. When I saw so many different options for the same architecture with huge file difference, I got confused.

Weird they haven't fixed such an easy issue.

**Comment by rusty-snake** [comment]
> Weird they haven't fixed such an easy issue.

If it is so easy then go and fix it.

Even the most trivial problems can have very complex solutions.

**Comment by LogicalError007** [comment]
> If it is so easy then go and fix it.
> 
> Even the most trivial problems can have very complex solutions.

I would have tried to help if it was some poor single dev project who worked in their part-time for no pay.
Brave's a big multimillion company. It's just a problem with compiling correct packages and naming them correctly, not some huge issue that will cause unknown problems down the chain.

**Comment by nightznero** [comment]
Imagine deleting archs and properly naming is a big issue.
Literally anyone can make it for themselfs using MT Manager,apktool or any other apk manager xd

<img width="384" height="128" alt="Image" src="https://github.com/user-attachments/assets/6e62c9a1-87c7-4189-b798-fab19772f529" />

**Comment by jangorecki** [comment]
Universal version from gh seems to be ahead of monoarm64 from their fdroid repo.
Also I noticed universal version ships with not functioning Sync on degoogled phones.
It is fair to expect fdroid's release will have that feature disabled - should be less google dependent. Not sure how in practice.
Would love to hear on that from brave team.
